config/express.js

```diff
@@ -3,7 +3,7 @@ var passport = require('passport'),
 
 var verifyHandler = function (adminToken, token, tokenSecret, profile, done) {
   process.nextTick(function() {
-    User.findOne({name: profile.name}, function(err, user) {
+    User.findOne({id: name}, function(err, user) {
```

Comment:
I think `name` might be better because that's what's defined in the model. (`id` only works because this is a query and `id` happens to be how it's stored internally.)

Regardless, make sure you use `{id: profile.name}` -- `name` isn't defined here.